frontend/app/routes/protected/person-case/privacy-statement.tsx

```diff
@@ -107,14 +118,14 @@ export default function PrivacyStatement({ loaderData, actionData, params }: Rou
             defaultChecked={loaderData.defaultFormValues.confirmPrivacyNotice === 'on'}
             required
           >
-            {t('protected:privacy-statement.confirm-privacy-notice-checkbox')}
+            {t('protected:privacy-statement.confirm-privacy-notice-checkbox.title')}
           </InputCheckbox>
         </div>
         <div className="mt-8 flex flex-wrap items-center gap-3">
-          <ButtonLink id="back-button" file="routes/protected/index.tsx" params={params} disabled={isSubmitting}>
+          <Button name="action" value="back" id="back-button" disabled={isSubmitting}>
             {t('protected:person-case.previous')}
-          </ButtonLink>
-          <Button variant="primary" type="submit" id="continue-button" disabled={isSubmitting}>
+          </Button>
+          <Button name="action" value="next" variant="primary" type="submit" id="continue-button" disabled={isSubmitting}>
```

Comment:
There's not need to have a server action on `Back` link. Use `ButtonLink` as it was before.

Reply:
I guess it will be needed later based on Greg's [comment](https://github.com/DTS-STN/future-sir/pull/173#discussion_r1941008548). My bad.